ledger/error/BUILD.bazel

```diff
@@ -17,16 +17,9 @@ da_scala_library(
     tags = ["maven_coordinates=com.daml:error:__VERSION__"],
     visibility = ["//visibility:public"],
     deps = [
-        "//daml-lf/archive:daml_lf_archive_reader",
-        "//daml-lf/data",
-        "//daml-lf/engine",
-        "//daml-lf/language",
-        "//daml-lf/transaction",
-        "//daml-lf/validation",
         "//ledger-api/grpc-definitions:ledger_api_proto_scala",
         "//ledger/ledger-grpc",
         "//ledger/participant-integration-api:participant-integration-api-proto_scala",
```

Comment:
Isn't that one still problematic?

Reply:
Not really, it just pulls in some proto defs from the Index. It also seems that it the target compiles without it. Removed it